std/stdio.d

```diff
@@ -1368,6 +1368,49 @@ Throws: $(D Exception) if the file is not opened.
         w.put('\n');
     }
 
+    /**
+
+    Writes its arguments in text format to the file separated pairwise by a
+    space (or custom `separator`), followed by a newline (or custom `eol`).
+
+    Throws: $(D Exception) if the file is not opened.
+            $(D ErrnoException) on an error writing to the file.
+    */
+    void print(string separator = " ", string eol = "\n", S...)(S args)
```

Comment:
I don't like putting this inside `File`. It's yet another method that does something another method already does, except different. What about move this to module scope and take `File` as first argument, so that we can make use of UFCS?